support-workers/docs/error-handling.md

```diff
@@ -18,7 +18,7 @@ Most errors should be logged to Sentry so they can be investigated. However tran
 
 | Error type           | Description                                     | Retry?    |
 |----------------------|-------------------------------------------------|-----------|
-| Input parsing errors | Errors caused by invalid input JSON             | No        |
+| Input parsing errors | Errors caused by invalid input JSON             | Yes       |
```

Comment:
I didn't know it was supposed to fail - in practice it actually retries, which is very useful if you are deploying a change to the format - as you can deploy support-fronetend first and then support-workers later.